addons/mail/models/mail_mail.py

```diff
@@ -367,11 +382,13 @@ def _personalize_outgoing_body(self, body, partner=False, recipients_follower_st
             body = re.sub(_UNFOLLOW_REGEX, '', body)
         return body
 
-    def _prepare_outgoing_list(self, recipients_follower_status=None):
+    def _prepare_outgoing_list(self, mail_server=False, recipients_follower_status=None):
         """ Return a list of emails to send based on current mail.mail. Each
         is a dictionary for specific email values, depending on a partner, or
         generic to the whole recipients given by mail.email_to.
 
+        :param mail_server: mail_server that will be used to send the mails,
```

Comment:
Could mention the model, like done in other docstrings: `:param partner: <res.partner> recipient`